src/Extend/SitesLinkedToMagentoStores.php

```diff
@@ -20,4 +22,38 @@ public function findByMageRunCode($code)
     {
         return collect($this->sites)->get($code);
     }
+
+    protected function getSavedSites()
+    {
+        return Cache::rememberForever('statamic_sites', function () {
+            $sites = [];
+            $stores = Rapidez::getStores();
+            $configModel = config('rapidez.models.config');
+
+            foreach ($stores as $store) {
+                if (config('rapidez.statamic.sites.' . $store['code'] . '.attributes.disabled')) {
```

Comment:
Can't we safely say we always want this to be enabled? And otherwise maybe just have an array that includes the disabled stores?

Reply:
Changed this around, see https://github.com/rapidez/statamic/pull/81#issuecomment-2457255941